hack/testing/test-upgrade.sh

```diff
@@ -32,6 +52,26 @@ function dumpEvents() {
 }
 trap dumpEvents EXIT
 
+function createOldIndexPattern() {
+  echo "creating index with old pattern"
+
+  indexDate=`date +%Y.%m.%d`
+  genUUID=`uuidgen`
+
+  esPod=`oc get pods -l component=es -o name | sed "s,pod/,,"`
+
+  waitFor "[[ \"Running\" == \"\$(oc get pods -l component=es -o jsonpath='{.items[*].status.phase}')\" ]]" "$(( 3 * TIME_MIN ))" || return 1
+  [ -z "$esPod" ] && echo "Unable to find ES pod for recreating old index pattern" && return 1
+
+  # create an old index pattern
+  oc exec $esPod -- curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key \
+                         -XPUT "https://logging-es:9200/oldindex.${genUUID}.${indexDate}" -d '{ "settings": { "index": { "number_of_shards": 1, "number_of_replicas": 0 } } }'
+
+  # necessary?
+  oc exec $esPod -- curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key \
```

Comment:
we probably can remove this if you want... Im sure it isn't causing any harm though